utils/utils_test.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import (
+	"testing"
+
+	"_vendor/github.com/stretchr/testify/assert"
```

Comment:
I recommend (1) moving ToSnake() back into the original package instead of creating utils, and (2) writing a unit test that just uses the testing package from the standard library.